src/pages/workspace/WorkspaceResetBankAccountModal.js

```diff
@@ -12,9 +13,16 @@ const propTypes = {
     /** Reimbursement account data */
     reimbursementAccount: ReimbursementAccountProps.reimbursementAccountPropTypes.isRequired,
 
+    /** Callback when the user confirms resetting the workspace bank account */
+    onConfirm: PropTypes.func,
+
     ...withLocalizePropTypes,
 };
 
+const defaultProps = {
+    onConfirm: null,
```

Comment:
or you can default to `() => {}` and then no need for `if (props.onConfirm) {` below